scripts/xe-enable-ipv6

```diff
@@ -23,33 +23,39 @@ fi
 . /etc/xensource-inventory
 
 if [ $1 = "enable" ]; then
-	rm /etc/modprobe.d/disable-ipv6 
-	
+	rm /etc/modprobe.d/disable-ipv6.conf
+
 	# Enable IPv6 networking
 	sed -i '/^NETWORKING_IPV6=/d' /etc/sysconfig/network
 	sed -i '/^IPV6_AUTOCONF=/d' /etc/sysconfig/network
 	echo "NETWORKING_IPV6=YES" >> /etc/sysconfig/network
 	echo "IPV6_AUTOCONF=NO" >> /etc/sysconfig/network
 	chkconfig ip6tables on
-	
+
+	sysctl -q -w "net.ipv6.conf.all.disable_ipv6=0"
+	sysctl -q -w "net.ipv6.conf.default.disable_ipv6=0"
```

Comment:
it seems this command overrides the entries but it's lost at reboot.Is there a way to write in `/etc/sysctl.d/90-net.conf`?

Reply:
I don't think there is a particularly nice way, but you could use a combination of `grep`, `sed` and `>>`

Reply:
Yes I was afraid of that. But i didn't find anything offering another solution. ^^

Reply:
Would it make sense to move the ipv6 settings to `/etc/sysctl.d/91-net-ipv6.conf`. So the whole file can be written instead of being a partial update?

Reply:
No idea, I'm okay to do that if you and everyone in the xapi team are in favor. :)

Reply:
Could you please go ahead with the new config file?

Reply:
On it! :)

Reply:
Done, should i check that the file alreydy exists? and also should i remove the settings from `90-net.conf`? Or should it be done elsewhere.